src/protagonist/API.Tests/Integration/DeliveryChannelTests.cs

```diff
@@ -18,13 +26,38 @@ namespace API.Tests.Integration;
 [Collection(CollectionDefinitions.DatabaseCollection.CollectionName)]
 public class DeliveryChannelTests : IClassFixture<ProtagonistAppFactory<Startup>>
 {
+    private static readonly IEngineClient EngineClient = A.Fake<IEngineClient>();
+    private readonly ControllableHttpMessageHandler httpHandler;
     private readonly HttpClient httpClient;
     private readonly DlcsContext dbContext;
+    private readonly string[] fakedAvPolicies =
+    {
+        "video-mp4-480p",
+        "video-webm-720p",
+        "audio-mp3-128k"
+    };
     
     public DeliveryChannelTests(DlcsDatabaseFixture dbFixture, ProtagonistAppFactory<Startup> factory)
     {
         dbContext = dbFixture.DbContext;
-        httpClient = factory.ConfigureBasicAuthedIntegrationTestHttpClient(dbFixture, "API-Test");
+        httpHandler = new ControllableHttpMessageHandler();
+        httpClient = factory
```

Comment:
There's an overload of `ConfigureBasicAuthedIntegrationTestHttpClient` that takes a delegate to customise the webAppFactory, saves repeating the Auth stuff.

```cs
httpClient = factory.ConfigureBasicAuthedIntegrationTestHttpClient(dbFixture, "API-Test",
            f => f.WithLocalStack(storageFixture.LocalStackFixture));
```